src/map/status.cpp

```diff
@@ -12533,7 +12526,7 @@ int status_change_start(struct block_list* src, struct block_list* bl,enum sc_ty
 			val2 = min(10*val1, 99); // % damage received reduced from 10 * skill lvl up to 99%
 			break;
 		case SC_VIGOR: {
-				uint8 hp_loss[10] = { 15, 14, 12, 11, 9, 8, 6, 5, 3, 2 };
+				uint8 hp_loss[10] = { 100, 90, 80, 70, 60, 50, 40, 30, 20, 10 };
 
 				val2 = hp_loss[val1- 1];
 			}
```

Comment:
```suggestion
			val2 = 10 * val1;
```
Sorry, Github wouldn't let me do one review suggestion since there is a deleted line between. If the level differences have no special values between like before, we can just do a simple formula!